File: Android/dokit/src/main/java/com/didichuxing/doraemonkit/kit/crash/CrashInfo.java
package com.didichuxing.doraemonkit.kit.crash;

import java.io.Serializable;

/**
 * Created by wanglikun on 2019-06-12
 */
public class CrashInfo implements Serializable {
    public final Throwable tr;

    public final long time;

    public CrashInfo(Throwable tr, long l) {
        this.tr = tr;
        this.time = l;
    }
}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/crash/CrashInfo.kt
package com.didichuxing.doraemonkit.kit.crash

import java.io.Serializable

/**
 * Created by wangxueying on 2020-06-30
 */
class CrashInfo : Serializable {
    var tr: Throwable? = null
    var time: Long? = null
}